tests/pubsub/test_gossipsub.py

```diff
@@ -148,7 +149,7 @@
     await connect(hosts[index_alice], hosts[index_bob])
 
     # Wait 3 seconds for heartbeat to allow mesh to connect
```

Comment:
Dangling magic number here.
```suggestion
     # Wait for heartbeat to allow mesh to connect
```